.travis.yml

```diff
@@ -0,0 +1,16 @@
+before_install:
+    - sudo apt-get -qq update
+    - sudo apt-get install -y groff
+
+script:
+    - make -C protocols/ all html
```

Comment:
Isn't it easier to do
`mkdir build && cd build`
`make -C ../protocols/ all html`

Reply:
`-C` will change directory, you still need to copy the output files after build.